docs/source/reading_plans.rst

```diff
@@ -1,123 +1,237 @@
+   :width: 32%
+
+.. |reading_plan_daily_readings| image:: /images/reading_plan_list.png
+   :width: 32%
+
+Abandon Reading Plan
+--------------------
+
+Press the vertical 3-dot menu from the `Daily Reading` screen, and click `Reset`.
+Next time you return to the same `Reading Plan`, the progress will be reset to day 1.
+
+Switch Between Reading Plans
 ----------------------------
 
+From the `Daily Reading` screen, click the first button in the top lef of the
+toolbar. This will let you select a different reading plan. You can return to
+the original reading plan and your progress will remain where you left it.
+In other words, it is possible to use several reading plans simultaneously.
+
+Preview Reading Plans or Go To Specific Day
+-------------------------------------------
+
+When in the `Daily Reading` screen, click the second button in the top left of
+the toolbar. You will see a list of daily readings. Clicking on a daily reading
+will bring you to `Daily Reading` screen for that specific day.
+
+Changing Reading Plan Start Date
+--------------------------------
+
+If you started the readings on a previous day and have just switched to |app|,
+then select the vertical 3-dot menu from the `Daily Reading` screen and
+click `Set Start Date...` to pick a specific start date or click `Set as current day...`
+to set the start date of the reading plan to today's date.
+
+Set Specific Reading Plan Day to Today's Date
+---------------------------------------------
+
+This is useful if you have been reading outside of |app|. To
+move the current reading plan day to today's date:
+
+#. Open the `Daily Reading` screen for the reading plan you would like to adjust
+#. Go to the specific day of the reading plan you want to set to today's date.
+#. Click the top right 3-dot menu
+#. Click `Set as current day`.
+
+.. note::
+
+    For |app| Versions 3.3 and earlier: Go to the day prior to the desired day
+    as described above, click  `Menu -> Done` (not the normal Done button at the bottom)
+    this bypasses checks and marks the current day as Done and the next day will become
+    the current day.
+
 Custom Reading Plans
-----------------------------
-It is possible to create a custom reading plan for use with AndBible.
+====================
+
+It is possible to create your own custom reading plan for use with |app|.
 
-Beginning in version 4.0 (September 2021), there is a new module format that can be used for reading plans. The .properties file remains the same, but the path in jsword/readingplan no longer works on Android 10+, so we have made a new way that reading plans can be imported like other commentaries and Bibles, in .zip format. The jsword/readingplan path will continue to work for Android versions lower than 10.
+.. note::
 
-The .zip format
+    If you create a reading plan that you think would be useful for other |app| users,
+    feel free to share it in our `Matrix <https://matrix.to/#/#andbible:matrix.org>`_
```

Comment:
Maybe Matrix / Telegram and then link to https://github.com/AndBible/and-bible/wiki/Support which has the specific links & instructions to these communication channels?